extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueLightDiscoveryService.java

```diff
@@ -74,6 +77,9 @@
     private final Logger logger = LoggerFactory.getLogger(HueLightDiscoveryService.class);
 
     private static final int SEARCH_TIME = 10;
+    private static final int DISCOVERY_SCAN_DELAY = 0;
+
+    private ScheduledExecutorService scheduler;
```

Comment:
The SonosDiscoveryService uses
`
private final ExecutorService pool = ThreadPoolManager.getPool(getClass().getSimpleName());
pool.submit(() -> { ... }
`